keyboards/kyria/rev1/config.h

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2019 Thomas Baart <thomas@splitkb.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include QMK_KEYBOARD_CONFIG_H
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```

Comment:
You may want to change this to something else,

Reply:
I don't have a vendor ID or product ID yet. Is there a recommended way to uniquely define that?

Reply:
Right now, unless you're able to get something, it's `0xFEED` for the vender ID and then basically whatever for the product ID.   And ideally, something that isn't being used already. 

Otherwise, there isn't anything official here, unless you've actually registered for one.